project/app/components/SearchForm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Form, Field } from '@availity/form';
+import { Button, Card } from 'reactstrap';
+import * as yup from 'yup';
+import { useAppStore } from '@/hooks';
+
+const schema = yup.object().shape({
+  memberId: yup
+    .string()
+    .isRequired(true, 'This Field is Required.')
+    .matches(/^\d{8}$/, 'Member ID must be 8 digits.'),
+  zipCode: yup
+    .string()
+    .isRequired(true, 'This Field is Required.')
+    .matches(/^\d{5}(?:-\d{4})?$/, 'Valid Zip Code Formats: 12345 or 12345-6789'),
+});
+
+async function stall(stallTime = 3000) {
+  await new Promise(resolve => setTimeout(resolve, stallTime));
+}
+
+const SearchForm = () => {
+  const { memberId, zipCode, setLoading, setMemberInfo } = useAppStore(store => ({
+    memberId: store.memberInfo ? store.memberInfo.memberId : '',
```

Comment:
just a style preference but I'm thinking use optional chaining here instead so people get used to it?